psydac/linalg/stencil.py

```diff
@@ -1410,14 +1415,18 @@ def diagonal(self, *, inverse = False, out = None):
         diag = self._data[diagonal_indices]
         data = out._data if out else None
 
+
```

Comment:
Minor correction.. We should not use two whitelines inside of a class definition, but only to separate two classes from each other (see PEP-8 for further information).
```suggestion
```

Reply:
Ok!